docs/proposals/27-project-group.md

```diff
@@ -0,0 +1,197 @@
+
+Also users can easily ignore those restrictions, by simply setting the `.spec.seedSelector` of a shoot to match a seed without the taint / outside the required geographical location.
+
+In order to enforce something like this, a new resource called [`SeedBinding`s](#seedbinding) will be introduced.
+
+Furthermore, [`ProjectGroup`s](#projectgroup) should decrease the work of Gardener operators by letting users group different projects into a single project group, where seed bindings will automatically be shared among them.
+
+### Goals
+
+The overarching goal of this GEP is to lay the foundational work to enable homogeneous sovereign cloud scenarios.
+
+- Enable projects to use some seeds exclusively by using the Taints and Tolerations feature ([`SeedBinding`s](#seedbinding))
+- Enhance projects so that shoot control planes can be restricted to certain geographical regions ([`SeedBinding`s](#seedbinding))
+- Share those, so called, seed bindings to different projects that logically belong together in a concept called [`ProjectGroup`s](#projectgroup)
+- Introduce new role for project group administrators, that are allowed to add projects to a project group
+
+### Non-Goals
+
+- Let users exclusively bind seeds on their own
+
+## Proposal
+
+#### ProjectGroup
+
+A ProjectGroup API will be introduced that abstracts certain information over one or more projects. With this, it will become possible to homogeneously share resources between projects.
+
+Such a ProjectGroup manifest looks like this:
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: ProjectGroup
+metadata:
+  name: group1
+spec:
+  namespace: gardenprojectgroup-group1
+  owner:
+    apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: john.doe@example.com
+  members:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: alice.doe@example.com
+    role: admin
+  projects:
+    - Project1
+    - Project2
+```
+
+The main idea behind `ProjectGroup`s is that they, like projects, have their own namespace.
+Every resource that is shareable (at first only `SeedBinding`s) in this namespace, will be shared to all projects in this project group.
+
+This sharing mechanism will be achieved by the `ProjectGroup` controller.
+It will copy the objects from the project group namespace to the namespaces of the projects that belong to the project group.
+
+When the object in the project group namespace is changed, the controller will immediately update the objects in the namespaces of the projects.
+
+To let a user in a project know that such an object is being kept in sync with an object in a project group, a `projectgroup.gardener.cloud/copied-from: <projectgroup-name>` label will be added to it.
+
+This copying feature can become useful for future use cases, for example for sharing infrastructure credentials across different projects, by copying `SecretBinding`s.
```

Comment:
Is the `SecretBinding` a good example? It already supports cross-namespace references to the actual infrastructure secret, all without copying.

Reply:
Sounds reasonable to me. The secret remains in the project group, but the secretBinding is copied to all projects attached to the project group. 
This way all projects would be able to use the secret for shoots etc., but the project members won't need access to the secret data.